app/assets/stylesheets/components/_option-select.scss

```diff
@@ -103,4 +103,44 @@
       }
     }
   }
+
+  // This is a temporary overwrite for checkboxes
+  // These styles should be removed in favour of small checkboxes from the Design System
+  .govuk-checkboxes__item {
+    min-height: 26px;
+    margin-bottom: 10px;
+    padding: 0 0 0 30px;
+  }
+
+  .govuk-checkboxes__item:last-child,
+  .govuk-checkboxes__item:last-of-type {
+    margin-bottom: 0;
```

Comment:
Is this necessary? Looking at it on https://finder-frontend-pr-742.herokuapp.com/component-guide/option-select there's already a rule in place that does this, not sure where it's coming from though.

Reply:
It's coming from the overwritten version. Doesn't need to be here too, I can remove this.